### PR TITLE
Update Octothorpe_105.srt

### PR DESCRIPTION
**Pull request by BohemianCoast**
Up to 17 minutes so far, 65 minutes of editing.

### CODE REVIEW

**Comment by johncoxon** [comment]
I don't think it's wise to have the speakers in the srts, since there are often speaker turns mid-line and I'm not sure it's trivial to edit where the srt breaks the lines themselves.

I think we'd be better off doing the corrected srt without the speaker turns, and then going through and adding speakers when we create the txt file.

What do you think?

File: Whisper (unedited AI transcripts)/Octothorpe_104.srt [hunk 1]
```diff
@@ -36,15 +36,15 @@
 
 10
 00:00:19,400 --> 00:00:36,800
- Hello everyone and welcome to the very 104th episode of Octa Thorpe.
+ Hello everyone and welcome to the very 104th episode of Octohorpe.
```
Comment (johncoxon) [inline Whisper (unedited AI transcripts)/Octothorpe_104.srt:39]:
```suggestion
 Hello everyone and welcome to the very 104th episode of Octothorpe.
```

File: Whisper (unedited AI transcripts)/Octothorpe_104.srt [hunk 2]
```diff
@@ -200,15 +200,15 @@
...
 
 52
 00:02:38,520 --> 00:02:40,520
- All of the idea is online.
+ All of Idea is online.
 
 53
 00:02:40,520 --> 00:02:46,440
- The recent type of issues are at Ephansines and the previous ones are Phalak.
+ The recent type of issues are at e-fanzines and the previous ones are at fanac.
```
Comment (johncoxon) [inline Whisper (unedited AI transcripts)/Octothorpe_104.srt:211]:
```suggestion
 The recent type of issues are at eFanzines and the previous ones are at FANAC.
```

**Review by johncoxon — APPROVED** [review]

**Comment by BohemianCoast** [comment]
I am still not sure I understand this, but I think it’s going to be ok with practice.

**Pull request merged**